### DISCUSSION TITLE
How to shift the drive cable of the linear drive on shoulder joint  to safe position

### DISCUSSION THREAD

**Discussion by jimingda** [robotology/community#634] [Q&A] [answered]
![icub](https://github.com/robotology/community/assets/7019692/c6240076-e551-4373-bbcb-16c3b4184b16)
![icub2](https://github.com/robotology/community/assets/7019692/ef287a8f-d5ab-457c-91b4-00692b59027c)

Hi all, as show in the pic, the drive cable of shoulder joint is on the edge seems to fall down the ankle, i don't know whether its normal, if its abnormal, how can i maintain it or shift it back to safer position?

Regards

**Comment by pattacini** [comment, marked as answer]
Hi @jimingda 

This is perfectly normal as the cables are routed through mutually orthogonal pulleys.
See for example https://icub-tech-iit.github.io/documentation/icub_tendons/shoulder_elbow/#3-left-elbow-cabling.

Next time, for HW-related issues, please consider opening a ticket on https://github.com/robotology/icub-tech-support/issues, as per the [guidelines](https://github.com/robotology/community#-hw-support).
